pkg/controller/controller.go

```diff
@@ -32,7 +32,7 @@ func New(services *services.Services) (*Controller, error) {
 		return nil, err
 	}
 
-	services.Router.PosStart(c.PostStart)
+	services.Router.PosStart(c.PostStart(services.MCPCatalogRefreshKick))
```

Comment:
nit: I probably would have made the `services.MCPCatalogRefreshKick` a field on the `Controller` type, but, since this is only used in the `PostStart` (which is only called once), I think this is fine.